htdocs/install/fileconf.php

```diff
@@ -221,7 +221,8 @@ class="minwidth300"
 				   name="main_url"
 				   value="<?php print $dolibarr_main_url_root; ?> "
 <?php if (!empty($force_install_noedit)) {
-	print ' disabled';
+	/* Behind a reverse proxy with https, we have to keep it writable */
```

Comment:
I suggest to replace test 
if !empty(force_install_no_edit)
With 
If !empty(force_install_no_edit) && force_install_no_edit != 3

So you can use value 3 in your config file...